### DISCUSSION TITLE
Where is my error - psalms in matins not appearing

### DISCUSSION THREAD

**Discussion by Padre-Anselmo** [DivinumOfficium/divinum-officium#3223] [Q&A] [answered]
Dear community,
I tried to make a file for Saint Wojciech (first class in Poland, 04-23.txt).
I wanted to do the Latin-breviary-file
...web/www/horas/Latin/Sancti/
I attach the file.
My question:
In the matins the psalms don't appear.
[04-23.txt](https://github.com/DivinumOfficium/divinum-officium/files/11419441/04-23.txt)

What did I do wrong.

Also:
What does the number after the "dot" mean (.1)? (in the [Rank] line of some of these files)
like
;;6.1;;

**Comment by mbab** [comment, marked as answer]
You should add psalm numbers:
```
Stabunt justi * in magna constántia advérsus eos qui se angustiavérunt, allelúja.;;1
;;2
;;3
V. Sancti et justi in Dómino gaudéte, allelúja.
R. Vos elégit Deus in hereditátem sibi, allelúja.
Ecce * quómodo computáti sunt inter fílios Dei, et inter sanctos sors illórum est, allelúja.;;4
;;5
;;6
V. Lux perpétua lucébit Sanctis tuis, Dómine, allelúja.
R. Et ætérnitas témporum, allelúja.
Lux perpétua * lucébit Sanctis tuis, Dómine, et ætérnitas témporum, allelúja.;;10
;;14
;;20
```

dot in ranks helps handling con(oc)currence

PS. avoid CAPITALIC words.

**Reply by Padre-Anselmo** [reply]
Thank you!
1. I could not find a detailed instruction about these things. All I can figure out, is what I can see, comparing existing files. Are there some more detailed instructions somewhere?
2. The numbers seem to work for Matutinum - wonderful. Thank you!
3. ... but not for Laudes (I tried to add ;;10 ;;11 ;;12 ... for the five antiphones).

Thank you in advance for help!

**Reply by Augustinus-Altovadensis** [reply]
For the Lauds, you probably need to separate the individual Psalms with a simple semicolon (;;10;11;12).
If it doesn't work the way I wrote it, try different variations. Also, unless the Antiphon is Alleluia, it might not work either.

Yes, I also had to figure out some things, as there is no comprehensive documentation, different people have different hacks and workarounds... But I'm glad this project exists, it makes my work so much easier!

**Reply by Padre-Anselmo** [reply]
Thank you, Jiří. I suppose, you are Czech. Perhaps "Štěpán Šrubař" has some more informations about detailed documentation? (he did the email standalone thing)

**Comment by mbab** [comment]
1. AFAIK there is no need to change psalms in Laudes as they always follow defined schemes.
2. Tried to add psalm number and it worked for me.
3. If Antiphonas for Vespers and Laudes are common use:
```
[Ant Laudes]
@:Ant Vespera
```
4. Look at [05-03r_pl](https://raw.githubusercontent.com/mbab/divinum-officium/propria_pl/web/www/horas/Latin/Sancti/05-03r-pl.txt)

**Reply by Padre-Anselmo** [reply]
Thank you for the wonderful example, full of very useful details (I had made a 'Regina Poloniae' pdf, but this file will be useful for me)

Still a question:
When we have several files for the same day in the Sanctus folder, for ex 05-03.txt and 05-03r.txt, or 05-03r_pl.txt, what will exactly happen? Which one will be used?

**Reply by mbab** [reply]
This one which is in calendar file.
Example from 1960 calendar:
```
*May*
05-01=05-01r=St Joseph the Worker Confessoris=6=
05-02=05-02=S. Athanasii Confessoris Ecclesiae Doctoris=3=
05-03=05-03r=Ss. Alexandri et sociorum Martyrum=1=
```
In case RP 4th line should be:
```
05-03=05-03r_pl=Beatæ Mariæ Virginis Reginæ Poloniæ=6=
```
_Note: in current stage all after second '=' is ignored._

**Reply by Padre-Anselmo** [reply]
I see. Thank you! Seems very logic.
